minio/api.py

```diff
@@ -1286,7 +1286,8 @@ def remove_incomplete_upload(self, bucket_name, object_name):
                 return
 
     def presigned_get_object(self, bucket_name, object_name,
-                             expires=timedelta(days=7)):
+                             expires=timedelta(days=7),
+                             response_headers=None):
```

Comment:
@balamurugana that would work here but due to dangers of passing a mutable as a default argument in python I think best practice is to pass `None`. See this SO for some discussion on that:
http://stackoverflow.com/questions/26320899/why-is-the-empty-dictionary-a-dangerous-default-value-in-python

Happy to change it though - just let me know.

I'm open to changing to `custom_headers` but I chose `response_headers` because it matches aws docs (http://docs.aws.amazon.com/AmazonS3/latest/API/RESTObjectGET.html).  Also the headers all follow the `repsonse-*` naming convention.